### PR TITLE
update paper and bibtex

### PR DESCRIPTION
**Pull request by jbytecode**
Due to the JOSS Submission: https://github.com/openjournals/joss-reviews/issues/4716

### CODE REVIEW

File: paper.md [hunk 1]
```diff
@@ -20,23 +20,23 @@ affiliations:
...
 
-This paper introduces **geostan**, an R package for analyzing spatial data using Bayesian inference. **geostan**'s spatial models were built using Stan, a platform for Markov chain Monte Carlo (MCMC) sampling [@gabry_2020; @stan_2022; @stan_2022b]. The primary focus of the package is areal data for socio-economic and health research. The package provides tools for a complete workflow for spatial regression and disease mapping, and has unique spatial measurement error (ME) models suitable for researchers using ACS estimates as covariates [@donegan_2021]. 
+This paper introduces **geostan**, an R [@rstats] package for analyzing spatial data using Bayesian inference. **geostan**'s spatial models were built using Stan, a platform for Markov Chain Monte Carlo (MCMC) sampling [@gabry_2020; @stan_2022; @stan_2022b]. The primary focus of the package is areal data for socio-economic and health research. The package provides tools for a complete workflow for spatial regression and disease mapping, and has unique spatial measurement error (ME) models suitable for researchers using ACS estimates as covariates [@donegan_2021].
```
Comment (ConnorDonegan) [inline paper.md:25]: The "chain" in "Markov chain Monte Carlo (MCMC)" is not usually capitalized---the other terms ("Markov" and "Monte Calro") are proper nouns

File: paper.md [hunk 2]
```diff
@@ -48,15 +48,15 @@ The package provides convenience functions for visualizing spatial patterns and
   - The Approximate Profile Likelihood (APLE) estimator for measuring spatial autocorrelation [@li_2007]
   - Effective sample size (ESS) calculation [@griffith_2005]
 
-These tools are provided for exploratory analysis, not 'cluster detection'; p-values are not provided. Graphics are created with **ggplot2** [@wickham_2016].
+These tools are provided for exploratory analysis, but not for detection of clusters so p-values are not provided. Graphics are created with **ggplot2** [@wickham_2016].
```
Comment (ConnorDonegan) [inline paper.md:51]: Can we change:

"These tools are provided for exploratory analysis, but not for detection of clusters so p-values are not provided"

to:

These tools are provided for exploratory analysis, but not for detection of clusters. For this and other reasons, p-values are not provided.

**Review by ConnorDonegan — CHANGES_REQUESTED** [review]
Thanks @jbytecode 

I just have two comments here

File: paper.md (hunk 2 above)
Comment (jbytecode) [reply paper.md:51]: @ConnorDonegan - you can first accept it as is and then fix them as you suggested. Please ping me when you've done with it.

**Comment by ConnorDonegan** [comment]
Thanks @jbytecode I merged and updated

**Pull request merged**